modules/test/playwright/tests/account-admin-web/accounts.spec.ts

```diff
@@ -937,32 +937,40 @@ test('LPS-101893 Account list should be paginated', async ({
 
 	await accountsPage.goto();
 
-	await setItemsPerPage(page, 4);
+	await setItemsPerPage(page, 20);
 
-	for (let i = 1; i < 6; i++) {
-		if (i < 5) {
+	for (let i = 1; i <= 21; i++) {
+		if (i < 21) {
 			await expect(
-				accountsPage.accountsTable.cell(`Account ${i}`)
+				accountsPage.accountsTable.cell(
+					`Account ${i.toString().padStart(2, '0')}`
+				)
 			).toBeVisible();
 		}
 		else {
 			await expect(
-				accountsPage.accountsTable.cell(`Account ${i}`)
+				accountsPage.accountsTable.cell(
+					`Account ${i.toString().padStart(2, '0')}`
+				)
 			).toHaveCount(0);
 		}
 	}
 
 	await nextPage(page);
 
-	for (let i = 1; i < 6; i++) {
-		if (i < 5) {
+	for (let i = 1; i <= 21; i++) {
+		if (i < 21) {
 			await expect(
-				accountsPage.accountsTable.cell(`Account ${i}`)
+				accountsPage.accountsTable.cell(
+					`Account ${i.toString().padStart(2, '0')}`
+				)
 			).toHaveCount(0);
 		}
 		else {
 			await expect(
-				accountsPage.accountsTable.cell(`Account ${i}`)
+				accountsPage.accountsTable.cell(
+					`Account ${i.toString().padStart(2, '0')}`
```

Comment:
String(i).padStart(2, '0')

Reply:
Or we can store the accounts in an array (like we did previously) and use accounts[i]